flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/utils/LogicalTypeChecks.java

```diff
@@ -120,6 +120,25 @@ public static boolean isProctimeAttribute(LogicalType logicalType) {
         return logicalType.accept(TIMESTAMP_KIND_EXTRACTOR) == TimestampKind.PROCTIME;
     }
 
+    public static boolean supportedTimeAttributeType(LogicalType logicalType) {
```

Comment:
nit: it would be nice to start methods with verbs e.g. `isTimeAttributeType` or `isWatermarkType` which is also shorter

Reply:
What about `canBeTimeAttributeType`?

Reply:
+1